src/Tribe/Views/V2/View.php

```diff
@@ -1498,7 +1498,7 @@ static function () {
 			'before_events'        => tribe( Advanced_Display::class )->get_before_events_html( $this ),
 			'after_events'         => tribe( Advanced_Display::class )->get_after_events_html( $this ),
 			'display_events_bar'   => $this->filter_display_events_bar( $this->display_events_bar ),
-			'disable_event_search' => tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ),
+			'disable_event_search' => tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) && apply_filters( 'tribe_apply_settings_on_event_manager_page', false ) ),
```

Comment:
We should use the same pattern I suggested here:
https://github.com/the-events-calendar/events-pro/pull/1857/files#r759356356